src/signatures/ed25519.rs

```diff
@@ -37,6 +37,10 @@ impl SecretKey {
 pub struct PublicKey(ed25519_zebra::VerificationKey);
 
 impl PublicKey {
+    pub fn verify(pk: &PublicKey, sig: &Signature, msg: &[u8]) -> bool {
```

Comment:
The first parameter `pk: &PublicKey` should be `self` so that we can use it the same way zebra is using it `pk.verify(sig, msg)`

Reply:
aight